0xProject/0x-monorepo
# Integrate Heap analytics into the website

## Description

I basically removed react-ga, included Heap (which is not available as a UMD module) and replaced all the calls to GA with calls to Heap. 

I increased the amount of data we are logging and improved the way we are logging the data since Heap provides a better / more expansive API.
https://docs.heapanalytics.com/reference#addeventproperties. 

I also fixed `onPageLoadAsync` which wasn't actually working.

## Testing instructions

Play around with the website and go to our heap dashboard (I can send invite) and see how things are going! The main reason we want this is for the SQL feature though.

## Types of changes




## Checklist:
